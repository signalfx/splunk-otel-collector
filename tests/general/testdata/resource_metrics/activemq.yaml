resource_metrics:
  - scope_metrics:
      - metrics:
          - name: counter.amq.TotalConnectionsCount
            type: IntMonotonicCumulativeSum
          - name: jmx_memory.committed
            type: IntGauge
