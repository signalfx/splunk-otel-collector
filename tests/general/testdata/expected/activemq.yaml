resourceMetrics:
  - resource: {}
    scopeMetrics:
      - metrics:
          - gauge:
              dataPoints:
                - asInt: "3801088"
                  attributes:
                    - key: dsname
                      value:
                        stringValue: value
                    - key: host
                      value:
                        stringValue: docker-desktop
                    - key: hostHasService
                      value:
                        stringValue: activemq
                    - key: plugin
                      value:
                        stringValue: GenericJMX
                    - key: plugin_instance
                      value:
                        stringValue: memory_pool-Compressed Class Space
                    - key: system.type
                      value:
                        stringValue: activemq
                  timeUnixNano: "1000000"
            name: jmx_memory.committed
          - gauge:
              dataPoints:
                - asInt: "12189696"
                  attributes:
                    - key: dsname
                      value:
                        stringValue: value
                    - key: host
                      value:
                        stringValue: docker-desktop
                    - key: hostHasService
                      value:
                        stringValue: activemq
                    - key: plugin
                      value:
                        stringValue: GenericJMX
                    - key: plugin_instance
                      value:
                        stringValue: memory_pool-Code Cache
                    - key: system.type
                      value:
                        stringValue: activemq
                  timeUnixNano: "1000000"
            name: jmx_memory.committed
          - gauge:
              dataPoints:
                - asInt: "7864320"
                  attributes:
                    - key: dsname
                      value:
                        stringValue: value
                    - key: host
                      value:
                        stringValue: docker-desktop
                    - key: hostHasService
                      value:
                        stringValue: activemq
                    - key: plugin
                      value:
                        stringValue: GenericJMX
                    - key: plugin_instance
                      value:
                        stringValue: memory_pool-PS Survivor Space
                    - key: system.type
                      value:
                        stringValue: activemq
                  timeUnixNano: "1000000"
            name: jmx_memory.committed
        scope: {}
